charts/dapr/charts/dapr_operator/values.yaml

```diff
@@ -16,6 +16,8 @@ fullnameOverride: ""
 
 runAsNonRoot: true
 
+serviceReconciler: true
```

Comment:
Would prefer to have a disable flag here instead of an enable flag.
Because the default behavior is enabled. So providing a flag to disable it makes it more consistent.

Reply:
Done